move/operators/sources/operators.move

```diff
@@ -18,10 +20,46 @@ module operators::operators {
     /// The main `Operators` struct storing the capabilities and operator IDs.
     public struct Operators has key {
         id: UID,
-        operator_ids: vector<ID>,
+        // number of operators should small and under the Sui vector limits
+        operators: VecSet<ID>,
+        // map-like collection of capabilities stored as Sui objects
         caps: Bag,
     }
 
+    // -----
+    // Error constants
+    // -----
+
+    /// When the operator is not found in the set of approved operators.
+    const EOperatorNotFound: u64 = 0;
+
+    /// When the capability is not found.
+    const ECapNotFound: u64 = 1;
+
+    // -----
+    // Events
+    // -----
+
+    /// Event emitted when a new operator is added.
+    public struct OperatorAdded has copy, drop {
+        operator_id: ID,
```

Comment:
use address and not ids

Reply:
See above. I still think we should use IDs